### ISSUE TITLE
OSC module's Command Tester does not update parameters when switching a command

### ISSUE THREAD

**Issue by sebescudie** [benkuper/Chataigne#202]
**Environment**
 - OS: Windows 11
 - Version 1.9.17b7

**Describe the bug**
When trying commands in an OSC module's Command Tester, switching commands does not update their parameters. The tester sticks to the parameters of the first command that was tested

**To Reproduce**
1. Create an OSC Module, add two template : `/foo` that sends a `float` and `/bar` that sends a `bool`
2. Go to the module's command tester, test `/foo` and then switch to `/bar` : you still have a `float parameter`

https://github.com/benkuper/Chataigne/assets/8690430/8b543cad-3f91-4263-a25a-c7ead0659e55

**Expected behavior**
The parameters should update and display those from the command I have selected

**Additional context**
None

**Comment by benkuper** [comment]
Yep, it tries to load the previous command data to make something "intelligent" and avoid reputting informations if the command is similar, but doing a poor job at it and basically reloading too much of the previous command. I disabled this feature for the next version

**Comment by sebescudie** [comment]
Noted, thanks!
